asks/request_object.py

```diff
@@ -244,6 +244,11 @@ async def make_request(self, redirect=False):
         if self.streaming:
             return None, response_obj
 
+        if 'connection' in asks_headers and asks_headers['connection'] == 'close' and self.sock._active:
+            self.sock._active = False
+            await self.sock.close()
```

Comment:
I wonder why the redirect case few lines above marks the socket as inactive but does not close it :thinking: I cannot see how it might be closed other than by a garbage collector. Maybe we should handle both cases in the same way and close sockets explicitly?

Reply:
Good catch 👍